pkg/compute/deploy.go

```diff
@@ -424,7 +424,7 @@ func createService(progress text.Progress, client api.Interface, name string, de
 		if strings.Contains(err.Error(), "Valid values for 'type' are: 'vcl'") {
 			return nil, "", errors.RemediationError{
 				Inner:       fmt.Errorf("error creating service: you do not have the Compute@Edge feature flag enabled on your Fastly account"),
-				Remediation: "See more at https://fastly.dev/learning/compute/#create-a-new-fastly-account-and-invite-your-collaborators",
+				Remediation: "See more at https://fastly.help/cli/ecp-feature",
```

Comment:
I think we concluded that most terminals will recognise this as a URL without the 'https://', so consider removing it?
```suggestion
				Remediation: "For more help with this error see fastly.help/cli/ecp-feature",
```